felix/dataplane/linux/int_dataplane.go

```diff
@@ -765,6 +766,26 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		dp.iptablesMangleTables = append(dp.iptablesMangleTables, mangleTableV6)
 		dp.iptablesFilterTables = append(dp.iptablesFilterTables, filterTableV6)
 
+		if config.RulesConfig.VXLANEnabled {
+			routeTableVXLAN := routetable.New([]string{"^vxlan.calico$"}, 6, true, config.NetlinkTimeout,
+				config.DeviceRouteSourceAddressIPv6, config.DeviceRouteProtocol, true, unix.RT_TABLE_UNSPEC,
+				dp.loopSummarizer, routetable.WithLivenessCB(dp.reportHealth))
+
+			vxlanManager := newVXLANManager(
+				ipSetsV6,
+				routeTableVXLAN,
+				"vxlan.calico",
```

Comment:
I'm not 100% sure, but I think we need to give it a different device name from the IPv4 interface, say "vxlan-v6.calico"?